themes/mongodb/src/js/componentThirdParty.js

```diff
@@ -20,8 +38,39 @@ export function initialize() {
 
     // Delighted
     !function(e,t,r,n,a){if(!e[a]){for(var i=e[a]=[],s=0;s<r.length;s++){var c=r[s];i[c]=i[c]||function(e){return function(){var t=Array.prototype.slice.call(arguments);i.push([e,t])}}(c)}i.SNIPPET_VERSION="1.0.1";var o=t.createElement("script");o.type="text/javascript",o.async=!0,o.src="https://d2yyd1h5u9mauk.cloudfront.net/integrations/web/v1/library/"+n+"/"+a+".js";var l=t.getElementsByTagName("script")[0];l.parentNode.insertBefore(o,l)}}(window,document,["survey","reset","config","init","set","get","event","identify","track","page","screen","group","alias"],"Dk30CC86ba0nATlK","delighted");
+
+    // Google Analytics
+    window.ga('create', 'UA-7301842-14', 'auto');
 }
 
-export function setup() {
-    window.delighted.survey();
+export function setup(fastNav) {
+    const project = document.body.getAttribute('data-project');
+    try {
+        const sampleFactor = SAMPLE_FACTORS[project] || 0.1;
+        window.delighted.survey({
+            minTimeOnPage: 180,
+            sampleFactor: sampleFactor
+        });
+    } catch (err) {
+        console.error(err);
+    }
+
+    // Update Google Analytics
+    try {
+        window.ga('set', 'page', location.pathname + location.query);
```

Comment:
What is `location.query`? I think we just want the pathname based on [these docs](https://developers.google.com/analytics/devguides/collection/analyticsjs/single-page-applications#handling_multiple_urls_for_the_same_resource).